src/parse.h

```diff
@@ -95,7 +95,6 @@ class ParameterFile{
 			std::cout << "Parameter <" << key << "> not found and no default value provided. Aborting." << std::endl;
 			throw;
 		}
-
```

Comment:
Same here, please keep things as they were before. I am somewhat picky about this, because it makes reviews more complicated, and most likely someone put effort into formatting things nicely, so we should not change them if there is not a purpose behind it.